conda_smithy/configure_feedstock.py

```diff
@@ -124,7 +124,7 @@ def _trim_unused_zip_keys(all_used_vars):
     """Remove unused keys in zip_keys sets, so that they don't cause unnecessary missing value
     errors"""
     groups = all_used_vars.get('zip_keys', [])
-    if groups and not isinstance(groups[0], list):
+    if groups and not any(isinstance(groups[0], obj) for obj in (list, tuple)):
```

Comment:
Just out of curiosity, why are they `list`s or `tuple`s?

Reply:
To be clear am just wondering about the type of collection as opposed to why there is one.

Reply:
because it can also be a string, so just checking if it's iterable isn't enough.  It might be better to invert it and check for string_types directly.

Reply:
To clarify just a bit more, this can be one of 2 things:

* a list of strings, with each string being a key from elsewhere in the file
* a list of lists of strings, same idea as before.

Ultimately, conda-build wants everything as the latter.  So, it tries to figure out if it has the former situation, and if so, it wraps the list of strings in another list.

Reply:
Sure was guessing it was probably `list`s of `list`s. Just wasn't sure where the `tuple` was (maybe) getting introduced.